### DISCUSSION TITLE
Text not found in the web element, how to get the text?

### DISCUSSION THREAD

**Discussion by GHRyunosuke** [GCuser99/SeleniumVBA#112] [Q&A] [answered]
I wonder in the following case, is there a way to get the text such as "2023", "2.20", "-11.29%"?

Unlike other normal cases, in the corresponding "div" element, could not find any text value (loaded as a picture??)...

**Comment by 6DiegoDiego9** [comment, marked as answer]
It seems to me likely a text (copy-pasted as a text) generated by a JS in some obscure way...

**Reply by GHRyunosuke** [reply]
Thank you @6DiegoDiego9 , i think i will just go and try some other websites.
